charts/artifact-hub/templates/hub_ingress.yaml

```diff
@@ -4,6 +4,9 @@ kind: Ingress
 metadata:
   name: {{ include "chart.resourceNamePrefix" . }}hub
   labels:
+  {{- if gt (len .Values.hub.ingress.extraLabels)  0 }}
```

Comment:
Could you use a `with` action instead of the `if`? (for consistency across the templates)

Reply:
Hello, done